cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java

```diff
@@ -173,9 +183,13 @@ void printMessage(String topic, MqttMessage message) {
         }
     }
 
-    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions) {
-        connectOptions.setUserName("admin");
-        connectOptions.setPassword("admin".toCharArray());
+    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions, String username, String password) {
```

Comment:
Depending on how many more parameters we may need to add in the future, we should maybe do this differently. It is not good if a method takes more than 5 or so parameters.

Reply:
Let's see what would happen in the future. Let's approve this one for now.